ruma-common/src/raw.rs

```diff
@@ -10,22 +10,20 @@ use serde::{
 };
 use serde_json::value::RawValue;
 
-use crate::EventContent;
-
-/// A wrapper around `Box<RawValue>`, to be used in place of event \[content\] \[collection\] types
-/// in Matrix endpoint definition to allow request and response types to contain unknown events in
-/// addition to the known event(s) represented by the generic argument `Ev`.
-pub struct EventJson<T> {
+/// A wrapper around `Box<RawValue>`, to be used in place of any type in the Matrix endpoint
+/// definition to allow request and response types to contain that said type represented by
+/// the generic argument `Ev`.
```

Comment:
This still doesn't make sense to me (we have a generic argument `T`, not `Ev`) and I might actually adjust the docs *now* to not mention the json-specific `RawValue` anymore. But I'll update the docs separately.

Reply:
Ok, sorry it wasn't any better!

Reply:
It was definitely better, I just want to change more things about it before publishing.